singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -793,6 +873,32 @@ inline SpinerEOSDependsRhoSie::SpinerEOSDependsRhoSie(const std::string &filenam
   }
 }
 
+
+template <typename Data>
+struct ShiftTransform {
+  PORTABLE_INLINE_FUNCTION
+  ShiftTransform(const Data& data) : data_(data) {}
+
+private:
+  const Data data_;
```

Comment:
We don't want this to be `const`.
```suggestion
  Data data_;
```
This was a lesson I had to learn myself since I had been hearing that you should make everthing `const`, and here it seems like you want to be absolutely sure you never modify the data, right?

Well the issue is that you implicitly delete the copy-assignment and move-assignment operators for the class. We _may_ want to be able to copy this class around (especially if we copy the Spiner class), so this can't be `const` unfortunately.